tests/integration/checkpoint.bats

```diff
@@ -144,6 +144,35 @@ function simple_cr() {
 	simple_cr
 }
 
+@test "checkpoint --pre-dump (with invalid parent path)" {
+
+	setup_pipes
+	runc_run_with_pipes test_busybox
```

Comment:
As you don't really checkpoint or restore anything, there's no need to do `setup_pipes` / `runc_run_with_pipes` / `check_pipes`. A simple
```bats
        runc run -d --console-socket "$CONSOLE_SOCKET" test_busybox
        [ "$status" -eq 0 ]
```
should be enough.